Modules/vc-module-catalog/VirtoCommerce.CatalogModule.Web/Module.cs

```diff
@@ -169,9 +169,19 @@ public void Initialize(IServiceCollection serviceCollection)
             serviceCollection.Configure<AuthorizationOptions>(configure =>
             {
                 configure.AddPolicy(typeof(ExportableProduct).FullName + "ExportDataPolicy", exportPolicy);
+                configure.AddPolicy(typeof(ExportableCatalogFull).FullName + "ExportDataPolicy", exportPolicy);
             });
 
             #endregion
+
```

Comment:
Please rebase on the latest export/dev and apply new factory generation scheme.

Reply:
OK. Done

Reply:
In new scheme registrations like below are not needed, you just need to register factory